udata/core/dataset/api.py

```diff
@@ -457,7 +457,7 @@ def put(self, dataset, rid):
         if resource.filetype == 'file':
             form._fields.get('url').data = resource.url
         form.populate_obj(resource)
-        resource.modified = datetime.now()
+        dataset.update_resource(resource)
```

Comment:
will the the resource be updated twice? once in populate_obj, once in update_resource?